File: src/main/java/com/api/booklog/response/BookmarkResponse.java
package com.api.booklog.response;

import lombok.Getter;
import lombok.ToString;


@Getter
@ToString
public class BookmarkResponse {
    private final long postId;
    private boolean status;

    public BookmarkResponse(long postId, boolean newStatus) {
        this.postId = postId;
        this.status = newStatus;
    }
    public void setStatus(boolean status) {
        this.status = status;
    }

}

File: front/src/entity/data/BookmarkResponse.ts
import { Expose } from 'class-transformer'

export class BookmarkResponse {
  @Expose()
  postId = 0

  @Expose()
  status = false

  constructor(postId: number, status: boolean) {
    this.postId = postId
    this.status = status
  }
}
